README.md

```diff
@@ -1347,6 +1347,7 @@ Software that does not fit in another section.
 - [OpenZiti](https://openziti.github.io/) - Fully-featured, self-hostable, zero trust, full mesh overlay network. Includes a 2FA support out of the box, clients for all major desktop/mobile OS'es. ([Source Code](https://github.com/openziti/ziti)) `Apache-2.0` `Go`
 - [Orchest](https://www.orchest.io/) - A new kind of IDE for Data Science. ([Demo](https://cloud.orchest.io), [Source Code](https://github.com/orchest/orchest)) `AGPL-3.0` `Docker`
 - [oTranscribe](https://github.com/oTranscribe/oTranscribe) - Free web app to take the pain out of transcribing recorded interviews. ([Demo](https://otranscribe.com/)) `MIT` `Javascript`
+- [Overseerr](https://overseerr.dev/) - Overseerr is a free and open source software application for managing requests for your media library. It integrates with your existing services, such as Sonarr, Radarr, and Plex!. ([Source Code](https://github.com/sct/overseerr)) `MIT` `Typescript`
```

Comment:
> The Language tag is the main server-side requirement for the software. Don't include frameworks or specific dialects.

https://docs.overseerr.dev/getting-started/installation the only supported installation method is Docker, please change the language/platform tag to `Docker`

> built to work with your existing Plex ecosystem
>Additions that depend on proprietary services outside the user's control must be marked ⚠.

Plex is proprietary, therefore please add the `⚠` marker